vrk-kpa/suomidigi
# SUOPA-719 Footer theming

To test: 
- `make fresh`
- Open the proto to a new browser window: https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/7c1dc8eb-0b46-4aa5-b434-be0fbcd8fc0c/Suomidigi-etusivu-launch/
- Compare the footer area with the proto. Check that the mobile width works ok with the layout.
